packages/desktop/electron-builder-config.js

```diff
@@ -94,8 +106,8 @@ const prodConfig = () => ({
     publish: {
         provider: 'generic',
         url: 'https://dl.firefly.iota.org/',
```

Comment:
@rajivshah3 @MarkNerdi could we upload to a subfolder so we could use eg `https://dl.firefly.iota.org/shimmer/latest.yml` instead of  `https://dl.firefly.iota.org/shimmer.yml`, and upload the artifacts also to that subfolder, so the bucket is a bit more organized? Just a though here, not really important or a blocker for this PR to get merged 🙏🏼

Reply:
![image](https://user-images.githubusercontent.com/105642810/222662089-fd1dfad4-62bc-4864-9fc2-223f408e4db8.png)
i think it is possible if we change this location in the `build-desktop` file

Reply:
Lets look at this when we have more access to s3